app/services/experimentation/medication_reminder_service.rb

```diff
@@ -0,0 +1,68 @@
+# This is not a scientific experiment like active and stale patient experiments. It does not need or have human study
+# approval and we are not excluding patients under 18 years old. We are leveraging the experimentation framework to
+# request patients to visit clinics to pick up enough blood pressure medication to last the current covid spike.
+# We're using the experimentation framework for two reasons:
+# 1) we want most of the same functionality, including batched daily notification scheduling and the ability to track
+#    results by leveraging the experimentation and notification models
+# 2) this is an excellent opportunity to real-world test the experimentation framework
+module Experimentation
+  class MedicationReminderService
+    PATIENTS_PER_DAY = 10_000
+
+    class << self
+      def schedule_daily_notifications(patients_per_day: PATIENTS_PER_DAY)
+        return unless Flipper.enabled?(:experiment)
+
+        experiment = Experimentation::Experiment.find_by!(state: [:new, :running], experiment_type: "medication_reminder")
+        experiment.selecting_state!
+
+        eligible_ids = medication_reminder_patients(experiment)
```

Comment:
We should probably pass through the `patients_per_day` to the `medication_reminder_patients` method to apply the limit there in SQL -- no reason to pass back thousands of patients if we just need 100 for the first batch.

Reply:
I think we didn't do that in the A/B because we wanted to randomize before limiting. But that doesn't really matter here, so you're probably right about that.